packages/expo-video/src/VideoView.types.ts

```diff
@@ -10,6 +10,16 @@ import type { VideoPlayer } from './VideoPlayer.types';
  */
 export type VideoContentFit = 'contain' | 'cover' | 'fill';
 
+/**
+ * Describes the type of the surface used to render the video.
+ * - `surfaceView`: Uses the SurfaceView to render the video. This value should be used in the majority of cases. Provides significantly lower power consumption, better performance, and more features.
```

Comment:
```suggestion
 * - `surfaceView`: Uses the `SurfaceView` to render the video. This value should be used in the majority of cases. Provides significantly lower power consumption, better performance, and more features.
```

Let's use inline highlight for API functions/components.